src/core/util/log.cc

```diff
@@ -64,12 +65,62 @@ namespace kovri
 {
 namespace core
 {
+namespace logging = boost::log;
+namespace expr = boost::log::expressions;
+namespace attrs = boost::log::attributes;
+
+
+std::string SetColor(
+    logging::value_ref<logging::trivial::severity_level,
+    logging::trivial::tag::severity> const& severity, bool color)
+{
+  if (!color)
+  {
+    return "";
+  }
+  else if (severity)
+  {
+    switch (severity.get())
+    {
+      case logging::trivial::fatal:
+      case logging::trivial::error:
+        return "\033[31m"; // Red
+      case logging::trivial::info:
+        return "\033[32m"; // Green
+      case logging::trivial::warning:
+        return "\033[33m"; // Yellow
+      case logging::trivial::debug:
+        return "\033[34m"; // Blue
+      case logging::trivial::trace:
+        return "\033[36m"; // Cyan
+    }
```

Comment:
This could be simplified with:

```diff
diff --git a/src/core/util/log.cc b/src/core/util/log.cc
index 1aba1994..2dfce01a 100644
--- a/src/core/util/log.cc
+++ b/src/core/util/log.cc
@@ -74,28 +74,28 @@ std::string SetColor(
     logging::value_ref<logging::trivial::severity_level,
     logging::trivial::tag::severity> const& severity, bool color)
 {
-  if (!color)
+  if (color)
   {
-    return "";
-  }
-  else if (severity)
-  {
-    switch (severity.get())
+    if (severity)
     {
-      case logging::trivial::fatal:
-      case logging::trivial::error:
-        return "\033[31m"; // Red
-      case logging::trivial::info:
-        return "\033[32m"; // Green
-      case logging::trivial::warning:
-        return "\033[33m"; // Yellow
-      case logging::trivial::debug:
-        return "\033[34m"; // Blue
-      case logging::trivial::trace:
-        return "\033[36m"; // Cyan
+      switch (severity.get())
+      {
+        case logging::trivial::fatal:
+        case logging::trivial::error:
+          return "\033[31m"; // Red
+        case logging::trivial::info:
+          return "\033[32m"; // Green
+        case logging::trivial::warning:
+          return "\033[33m"; // Yellow
+        case logging::trivial::debug:
+          return "\033[34m"; // Blue
+        case logging::trivial::trace:
+          return "\033[36m"; // Cyan
+      }
     }
+    return "\033[0m"; // Reset
   }
-  return "\033[0m"; // Reset
+  return "";
 }
 
 logging::formatter GetFormat(bool color)
```

Reply:
Applied, thanks